cve_bin_tool/output_engine/console.py

```diff
@@ -19,9 +19,38 @@
 from ..util import ProductInfo
 
 
+def format_version_info(versions):
+    """
+    formats version info to desirable output
+
+    versions = ['start_including', 'start_excluding', 'end_including', 'start_excluding']
+    Example:
+        format_version_info(['2.2.8', '', '2.2.11', '']) => "[2.2.8 - 2.2.11]"
+        format_version_info(['', '2.2.8', '', '2.2.11']) => "(2.2.8 - 2.2.11)"
+        format_version_infor([]) => "-"
+
+    Reference for Interval terminologies: https://en.wikipedia.org/wiki/Interval_(mathematics)
+    """
+
+    empty_range = ["", ""]
+    if versions:
+        affected_version = {
+            "including": [versions[0], versions[2]],
+            "excluding": [versions[1], versions[3]],
+        }
+        if affected_version["including"] is not empty_range:
```

Comment:
```suggestion
        if affected_version["including"] != empty_range:
```
Same for the next `if`